gcip/addons/python/jobs.py

```diff
@@ -177,8 +178,11 @@ def twine_upload(
         ],
     )
     job.add_variables(
-        TWINE_REPOSITORY_URL=repository_url,
-        TWINE_USERNAME=user,
-        TWINE_PASSWORD=varname_password,
+        TWINE_USERNAME=f"${twine_username_env_var}",
+        TWINE_PASSWORD=f"${twine_password_env_var}",
```

Comment:
I think this is a security issue. 
All `generated-config.yml` files are uploaded as Artifacts to Gitlab. 
I think a better way of handling this, would be to let the user handle how the secrets are received. Only allow Environment Variables within job not as job var.

Reply:
Not the values should be provided but only the names of the environment variables containing the values (`...env_var`)